data-import/src/main/resources/folijet/data-import/features/data-import-integration.feature

```diff
@@ -9203,6 +9203,125 @@ Feature: Data Import integration tests
     And match containsDuplicatesOfFields(overlayParsedRecord.content.fields, ['006', '007', '008']) == false
 
+  Scenario:  MODDATAIMP-1031 Modify action to remove 999 field and create Instance
+    * print 'Modify action to remove 999 field and create Instance'
+
+    # Create MARC-to-MARC mapping profile
+    Given path 'data-import-profiles/mappingProfiles'
+    And headers headersUser
+    And request
+      """
+      {
+        "profile": {
+          "name": "MODDATAIMP-1031 remove 999 field",
+          "description": "",
+          "incomingRecordType": "MARC_BIBLIOGRAPHIC",
+          "existingRecordType": "MARC_BIBLIOGRAPHIC",
+          "mappingDetails": {
+            "name": "marcBib",
+            "recordType": "MARC_BIBLIOGRAPHIC",
+            "mappingFields": [],
+            "marcMappingDetails": [
+              {
+                "order": 0,
+                "action": "DELETE",
+                "field": {
+                  "field": "999",
+                  "indicator1": "*",
+                  "indicator2": "*",
+                  "subfields": [
+                    {
+                      "subfield": "*"
+                    }
+                  ]
+                }
+              }
+            ],
+            "marcMappingOption": "MODIFY"
+          },
+          "hidden": false,
+        },
+        "addedRelations": [],
+        "deletedRelations": []
+      }
+      """
+    When method POST
+    Then status 201
+    * def marcToMarcMappingProfileId = $.id
+
+    # Create action profile for MODIFY MARC bib
+    * def mappingProfileEntityId = marcToMarcMappingProfileId
+    Given path 'data-import-profiles/actionProfiles'
+    And headers headersUser
+    * def profileAction = 'MODIFY'
+    * def folioRecord = 'MARC_BIBLIOGRAPHIC'
+    * def userStoryNumber = 'MODDATAIMP-1031'
+    * def folioRecordNameAndDescription = folioRecord + ' action profile for ' + userStoryNumber
+    And request read('classpath:folijet/data-import/samples/samples_for_upload/create_action_profile.json')
+    When method POST
+    Then status 201
+    * def marcBibActionProfileId = $.id
+
+    # Create job profile - Create Instance, Holdings and Item
+    * def defaultJobProfileId = 'fa45f3ec-9b83-11eb-a8b3-0242ac130003'
+    Given path 'data-import-profiles/jobProfiles'
+    And headers headersUser
+    And request
+      """
+      {
+        "profile": {
+          "name": "MODDATAIMP-1031: Remove 999 and create instance",
+          "description": "",
+          "dataType": "MARC"
+        },
+        "addedRelations": [
+          {
+            "masterProfileId": null,
+            "masterProfileType": "JOB_PROFILE",
+            "detailProfileId": "#(marcBibActionProfileId)",
+            "detailProfileType": "ACTION_PROFILE",
+            "order": 0
+          },
+          {
+            "masterProfileId": null,
+            "masterProfileType": "JOB_PROFILE",
+            "detailProfileId": "#(defaultJobProfileId)",
```

Comment:
@dmytrokrutii  I think that there should be not JobProfileId. There should be an action profile id, because deatilProfileType is ACTION_PROFILE.